trillian/testdata/Makefile

```diff
@@ -111,6 +111,29 @@ distclean: clean
 	rm -f $(SERVER_PUBKEYS) fake-ca.cert int-ca.cert int-ca.csr.pem
 	rm -f $(LEAF_CERTS) $(LEAF_CSRS) $(LEAF_CHAINS)
 
+
+# We also use a second fake CA as a trust root for CT tests. This is its private key.
+CA_1_PRIVKEY=fake-ca1.privkey.pem
+CA_1_PWD=ahenny
+
+# Corresponding Leaf certificates.
+LEAF_1_PRIVKEY=leaf-1.privkey.pem
+LEAF_1_PWD=louth
+
+# Fake Root CA 1
+fake-ca-1.cert: $(CA_1_PRIVKEY) fake-ca.cfg
+	openssl req -new -x509 -config fake-ca.cfg -set_serial 0x0406efac -days 3650 -extensions v3_ca1 -inform pem -key $(CA_1_PRIVKEY) -passin pass:$(CA_1_PWD) -out $@
```

Comment:
Can we have a different subject name for the new root cert, to reduce the chances of confusion

Reply:
What do you mean by subject name? ca-1 to ca-B or...?

Reply:
I mean the subject name in the cert, which gets pulled from the `fake-ca.cfg` file; both `fake-ca.cert` and `fake-ca-1.cert` have `Subject: C=GB, O=Google, OU=Eng, L=London, ST=London, CN=FakeCertificateAuthority`

Reply:
It's hardly a big deal though - fine as is if that's easier.